### DISCUSSION TITLE
Not able to find new UI Kit for PrimeVue 4

### DISCUSSION THREAD

**Discussion by fred104** [primefaces/community#3236] [Figma UI Kit] [answered]
Hi, we are currently using PrimeVue 4 in our product. However, the UI mockups provided by our designer differ from the PrimeVue 4 components. It seems our designer could only find the PrimeVue 3 UI Kit for their work, which is odd because, according to this [link](https://github.com/orgs/primefaces/discussions/2835), the PrimeVue 4 UI Kit download link should be available.

Could you please let us know where we can find the latest UI Kit for PrimeVue 4?

![01](https://github.com/user-attachments/assets/38fa88af-14fb-4a5f-b609-ecd5a720fc98)
![02](https://github.com/user-attachments/assets/d5eadb3f-c212-44b8-889b-e0241ac508df)

**Comment by cagataycivici** [comment]
UI Kit v3 is the latest version that is compatible with PrimeVue v4. The link is from a user post, not from the core team. 

UI Kit V2  -> PrimeVue v3
UI Kit V3  -> PrimeVue v4

**Comment by onursenture** [comment, marked as answer]
Hi, UI Kit 3.0.1 is the latest and compatible with PrimeVue 4. Their version numbers are not synced.

**Reply by fred104** [reply]
Are there any discrepancies between the components in Figma and those in PrimeVue? Or vice versa?

**Reply by onursenture** [reply]
We strive to ensure consistency. Unfortunately, we don't have a list of all inconsistencies for now.

Good news for the near future: The upcoming Figma-to-Code converter in our new designer tool will feature scheme-based validation, ensuring diff checking and automatic updates for consistency between the UI Kit and the code.

**Reply by fred104** [reply]
Got it. Thank you!
